charts/datadog/templates/_container-system-probe.yaml

```diff
@@ -48,6 +52,12 @@
       mountPath: /host/proc
       mountPropagation: {{ .Values.datadog.hostVolumeMountPropagation }}
       readOnly: true
+{{- if .Values.datadog.serviceMonitoring.enabled }}
```

Comment:
I don't exactly now how it works, but should we also check that `datadog.networkMonitoring.enable` is true too? 

can we have `serviceMonitoring` enable without `networkMonitoring`?

Reply:
> can we have serviceMonitoring enable without networkMonitoring?

Yes you can.